packages/fxa-settings/src/lib/auth-errors/en.ftl

```diff
@@ -4,6 +4,7 @@ auth-error-102 = Unknown account
 auth-error-103 = Incorrect password
 auth-error-105-2 = Invalid confirmation code
 auth-error-110 = Invalid token
```

Comment:
Would this string no longer be used? If so, we should remove it.

Reply:
I don't _think_ it would be used anymore. However, this is a quick patch that we are going to iterate on. Would it be okay to leave in place for now, in case there is a situation where it's used, and then clean this up later?